keanu-project/src/main/java/io/improbable/keanu/network/BayesianNetwork.java

```diff
@@ -63,6 +81,22 @@ public Vertex getVertexByLabel(VertexLabel label) {
         return vertices;
     }
 
+    public int getVertexCount() {
+        return vertexCount;
+    }
+
+    public Integer getVertexDegree(Vertex vertex) {
```

Comment:
Should we have a vertex Label version of this as well?  I see that being the most useful in the "general case"?

Reply:
In fact, if someone had the Vertex object itself, wouldn't they be best off just calling the Vertex method directly?